src/Examples/Advanced/CustomUnitModels/methanol_state_block_VLE.py

```diff
@@ -345,21 +350,23 @@ def rule_P_vap(self, j): # coefficient converts MPa to mmHg
             return log(7500.6168 * self.vapor_pressure[j]) == \
                 self._params.vapor_pressure_coeff[j, 'A'] - \
                 (self._params.vapor_pressure_coeff[j, 'B'] /
-                (self.temperature*100/pyunits.hectokelvin*pyunits.kelvin - self._params.vapor_pressure_coeff[j, 'C'])) # convert T to K
+                (pyunits.convert(self.temperature, pyunits.K) - \
+                self._params.vapor_pressure_coeff[j, 'C']))
         self.eq_P_vap = Constraint(self._params.component_list, rule=rule_P_vap)
 
     def _density_mol(self):
         self.density_mol = Var(self._params.phase_list,
                                units=pyunits.kmol/pyunits.m**3,
-                               doc="Molar density [kmol/m^3]")
+                               doc="Molar density")
 
         def density_mol_calculation(self, p):
             if p == "Vap":
                 return self.pressure == (self.density_mol[p] *
-                                         Constants.gas_constant/1000/pyunits.mol*pyunits.kmol *
-                                         self.temperature*100/pyunits.hectokelvin*pyunits.kelvin) # convert gas constant to MJ/kmol-K and T to K
+                                         pyunits.convert(Constants.gas_constant,
+                                           pyunits.MJ/pyunits.kmol/pyunits.K) *
+                                         pyunits.convert(self.temperature, pyunits.K))
             elif p == "Liq":
-                return self.density_mol[p] == 11.1  # kmol/m3 # dummy value
+                return self.density_mol[p] == 11.1  # dummy value
```

Comment:
You should apply the units to the constant itself for unit consistency. I.e. `11.1*pyunits.kmol/pyunits.m**3`